app/controllers/completed_exercises_controller.rb

```diff
@@ -0,0 +1,97 @@
+class CompletedExercisesController < ApplicationController
```

Comment:
I think if we can get through JUST this controller and its routes (making a route and then its method each time) than we have made a gigantic improvement to the course 👍